taker-frontend/src/components/Disclaimer.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+    Button,
+    Link,
+    Modal,
+    ModalBody,
+    ModalCloseButton,
+    ModalContent,
+    ModalFooter,
+    ModalHeader,
+    ModalOverlay,
+    Text,
+    useDisclosure,
+} from "@chakra-ui/react";
+import React, { useEffect, useState } from "react";
+
+// Disclaimer is a modal pop-up that opens up before you start using the app and
+// goes away after the user dismisses it. It displays the information we want
+// the user to know before they start using the software.
+export default function Disclaimer() {
+    const [ack, setAck] = useState<boolean>(false);
```

Comment:
It might get annoying that you always have to click this away. But I guess this is good enough for now :)

Reply:
good idea! I'll add the change for the next release